exasol/toolbox/nox/_format.py

```diff
@@ -26,7 +26,8 @@ def _pyupgrade(session: Session, files: Iterable[str]) -> None:
         "poetry",
         "run",
         "pyupgrade",
-        "--py38-plus",
+        "--py39-plus",
+        "--keep-runtime-typing",
```

Comment:
Needed as it keeps erroneously changing typing like Optional[str] to str | None, which isn't in python3.9, and I couldn't find a way to skip on a per line basis. There should be one :cry:

Reply:
Should be maybe a per project setting?

Reply:
However, you could maybe also use import future

Reply:
updated this to a per project setting, as pydantic + future does not work as it does for other cases.